scenarioo-client/app/scenario/scenario.html

```diff
@@ -30,7 +30,7 @@
             </span>
         </div>
     </div>
-    <div class="col-sm-4">
+    <div class="col-sm-2 pull-right">
```

Comment:
was this change really intended?

Reply:
it is just a question - maybe you can explain it to me.

Reply:
Yes it was intended to keep the two buttons (expand/collapse all & Hide/Showdetails) in the upper right corner. The buttons were moved because of the search bar.